Sources/MountebankSwift/Models/Stub/Stub.swift

```diff
@@ -50,4 +50,11 @@ public struct Stub: Equatable {
     public init(response: any StubResponse, predicates: [Predicate]) {
         self.init(responses: [response], predicates: predicates)
     }
+
+    /// Creates a stub with a single response and no predicates
+    ///
+    /// The response is always used
+    public init(response: any StubResponse) {
```

Comment:
Why not adding a default value of `[]` to the function above?

Reply:
Ease of use / improved documentation 

<img width="527" alt="Screenshot 2024-03-23 at 19 03 41" src="https://github.com/MountebankSwift/MountebankSwift/assets/1330668/278a6968-96ad-4023-b5c3-250cafae5e7b">

Instead of:

<img width="523" alt="Screenshot 2024-03-23 at 19 02 37" src="https://github.com/MountebankSwift/MountebankSwift/assets/1330668/6d92eb47-4760-4963-b6cc-29a8622c18fd">